utils/llfat_utils.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+template <typename real> struct su3_matrix {
+  std::complex<real> e[3][3];
+};
+template <typename real> struct su3_vector {
+  std::complex<real> e[3];
+};
+
+#ifdef __cplusplus
```

Comment:
do we ever include that in a C code?

Reply:
In the original tests, they were wrapped around the extern "C" https://github.com/lattice/quda/blob/cfac928a1fa97f407b993b08901b90e8b5ec0d0c/tests/llfat_reference.h

Reply:
Maybe for the MILC interface?

Reply:
Fixed